jobs/blog_validation_jobs.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validation Jobs for blog models"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform import models
+from jobs import base_jobs
+from jobs import blog_validation_errors
+from jobs.io import ndb_io
+
+import apache_beam as beam
+(blog_models,) = models.Registry.import_models([models.NAMES.blog, ])
+
+
+class BlogPostTitleUniquenessJob(base_jobs.JobBase):
```

Comment:
Since they are validation jobs, I think they can be included in the blog_validation.py file rather than creating a new file. This also has to restructured as a beam.DoFn with decorators. @brianrodri what do you think about this?

Reply:
I think this is a special case since uniqueness can only be determined by looking at every single blog in storage, so this is an appropriate "edge case" that deserves its own job.

Reply:
Oh ok, then I think it is fine. Thanks @brianrodri